### PR TITLE
[ws-daemon] Improve download options

### PR DESCRIPTION
**Pull request by aledbf**
## Description

Reduce verbosity while downloading content with aria2c and enabling retries after five seconds.

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-integration-tests=workspace with-large-vm=true
      Valid options are `all`, `workspace`, `webapp`, `ide`

### CODE REVIEW

**Review by sagor999 — APPROVED** [review]

File: components/ws-daemon/pkg/content/initializer.go [hunk 1]
```diff
@@ -399,6 +399,9 @@ func (rs *remoteContentStorage) Download(ctx context.Context, destination string
 
 	args := []string{
 		"-x16", "-j12",
+		"--retry-wait=5",
```
Comment (kylos101) [inline components/ws-daemon/pkg/content/initializer.go:402]: Good idea.

For posterity, from [options](https://aria2.github.io/manual/en/html/aria2c.html#options):
```
--retry-wait=<SEC>[¶](https://aria2.github.io/manual/en/html/aria2c.html#cmdoption-retry-wait)
Set the seconds to wait between retries. When SEC > 0, aria2 will retry downloads when the HTTP server returns a 503 response. Default: 0
```

**Review by kylos101 — APPROVED** [review]
yaas

**Comment by kylos101** [comment]
/hold 

Can you add a release note @aledbf ? :pray Will be good for admins to know we're making snapshot download more resilient.

**Comment by kylos101** [comment]
/hold

Tide is stuck

**Comment by kylos101** [comment]
/unhold

Tide is betterer

File: components/ws-daemon/pkg/content/initializer.go (hunk 1 above)
Comment (utam0k) [reply components/ws-daemon/pkg/content/initializer.go:402]: :eyes:

File: components/ws-daemon/pkg/content/initializer.go [hunk 2]
```diff
@@ -398,7 +398,9 @@ func (rs *remoteContentStorage) Download(ctx context.Context, destination string
 	tempFile.Close()
 
 	args := []string{
-		"-x16", "-j12",
+		"-s10", "-x16", "-j12",
```
Comment (kylos101) [inline components/ws-daemon/pkg/content/initializer.go:401]: @utam0k look here, `-s10` is also relevant

Comment (utam0k) [reply components/ws-daemon/pkg/content/initializer.go:401]: I'm wondering if the non-abbreviated one (`--split`) would be easier to understand the code than the abbreviated one, can I make a PR?

Comment (kylos101) [reply components/ws-daemon/pkg/content/initializer.go:401]: Honestly, even with the words, I have to read description to know what they do. :sweat: In other words, I'd still need the website, or help from the command, to know how to tune.

Comment (utam0k) [reply components/ws-daemon/pkg/content/initializer.go:401]: Unfortunately, I agree with you :+1:

**Pull request merged**